### ISSUE TITLE
Regression tests should be run using binaries in the source directory, not installed binaries

### ISSUE THREAD

**Issue by daglem** [steveicarus/iverilog#790]
./vvp_reg.pl uses installed binaries to run regression tests.

This can be quite confusing when one is trying to fix a bug in Icarus Verilog, and attempted bugfixes have no effect.

The regression tests should preferably be run using binaries in the source directory. Alternatively, it would help if it were documented that one has to run "make install" before running the regression tests.

**Comment by caryr** [comment]
I agree, now that the test suite is integrated into the source code we should have an option to run it using the uninstalled binaries. This would not be the default, but I would also want a make test target in the top level make file to handle configuring and making sure everything is built correctly and then run the tests.

We need to make sure all this works with the suffix option.

For my development work I always run make install, but I do see where you would like to keep the global versions stable.
